voting-contentApp/i18n/voting-app-language_en.properties

```diff
@@ -0,0 +1,13 @@
+#For Detail SubApp - The Form
+voting-app.detail.label = Poll Data
+voting-app.name.label = Title
+voting-app.description.label = Description
+voting-app.imageLink.label = Select Image
+voting-app.expiryDate.label = Expiry Date
+voting-app.question.label = Add Question
+voting-app.option1.label = Required Option 1
+voting-app.option2.label = Required Option 2
+voting-app.optionX.label = Extra Options
+voting-app.optionX.value.label =
```

Comment:
is this redundant property?

Reply:
No, it is not. The last statement with the empty value is for the porpuse of hiding default "voting-app.optionX.value.label" lable of each extra obtion.